docs/git-guide.md

```diff
@@ -1255,6 +1255,13 @@ $ git rebase --continue
 Applying: test change for docs
 ```
 
+Note: You can see recent changes made to a file by running the following
```

Comment:
I think this section should come after the "decide which edits should be applied" section. You should also explicitly state that this command is useful when you're comparing changes to a previous commit on upstream in order to not undo the changes made in the previous commit. Maybe "Tip" is also a bit more helpful?